<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FwProgressForm</class>
 <widget class="QWidget" name="FwProgressForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>238</width>
    <height>78</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background:		rgba(35,31,32,255);
	border: 6px solid rgba(0,174,239,255);
}  

QWidget:hover
{
	/*border:				3px solid rgb(0, 126, 126)*/
}        

/* ---------- QLabels ------------*/
QLabel
{
	background:		rgba(0,0,0,0);
	color: 				white;
	font: 				7pt &quot;Corbel&quot;;
	border:				none;
}

QLabel:hover
{
	border:				none;
}  

/*---------- QProgress Bar ---------*/
QProgressBar 
{
border: 2px solid rgb(25,25,25);
padding: 0px;
background: white;
 }

QProgressBar::chunk
{
background-color: rgba(0,174,239,255);
border-radius: 1px;
}

</string>
  </property>
  <widget class="QFrame" name="warningFrame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>238</width>
     <height>78</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame
{
	background:		rgba(35,31,32,255);
	border: 6px solid rgba(0,174,239,255);
}  </string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="warningSubFrame">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>8</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame 
{
background: rgba(0,0,0,0);
border:none;
}
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="found">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-1</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Corbel&quot;;</string>
     </property>
     <property name="text">
      <string>Updating Firmware...</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>181</width>
       <height>23</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="value">
      <number>20</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
